### PR TITLE
Keadm: add prompting for confirmation

### LINKED ISSUE #1992: Enhancement: add confirmation prompt when reset
**What would you like to be added/modified**:

Now, when we excute `keadm reset`, it will exit directly, without any confirmation prompt. Maybe we need a step for that.

**Why is this needed**:

To avoid a wrong input, or other reason not really want to reset, we need an intermediate step for confirmation. And it is also a best practice when we want to close or exit something.

For anyone who want to fix that, you can add something to `keadm/cmd/keadm/app/cmd/reset.go`.

### PR DESCRIPTION
**Pull request by daixiang0**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1992

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Keadm: add prompting for confirmation
```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/reset.go [hunk 1]
```diff
@@ -78,6 +81,18 @@ func NewKubeEdgeReset(out io.Writer, reset *types.ResetOptions) *cobra.Command {
...
+				fmt.Println("[reset] WARNING: Changes made to this host by 'keadm init' or 'keadm join' will be reverted.")
+				fmt.Print("[reset] Are you sure you want to proceed? [y/N]: ")
+				s := bufio.NewScanner(os.Stdin)
+				s.Scan()
+				if err := s.Err(); err != nil {
+					return err
+				}
+				if strings.ToLower(s.Text()) != "y" {
+					return fmt.Errorf("aborted reset operation")
+				}
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/reset.go:94]: Do we need to have some retry like 3 times here?

Comment (daixiang0) [reply keadm/cmd/keadm/app/cmd/reset.go:94]: Similar action as `kubeadm`, just add one prompting for confirmation.

**Pull request merged**